SSA/Projects/LLVMRiscV/Evaluation/.gitignore

```diff
@@ -0,0 +1,19 @@
+benchmarks/MLIR_single
+benchmarks/MLIR_bb0
+benchmarks/LEANMLIR_ASM
+benchmarks/LEANMLIR_ASM_opt
+benchmarks/LLC_ASM_globalisel
+benchmarks/LLC_ASM_selectiondag
+benchmarks/LLVM
+benchmarks/LLVMIR
+benchmarks/logs
+benchmarks/XDSL_ASM
+benchmarks/XDSL_no_casts
+benchmarks/XDSL_reg_alloc
+benchmarks/XDSL_opt_ASM
+benchmarks/XDSL_no_casts_opt
+benchmarks/XDSL_reg_alloc_opt
+mca-analysis/results
+mca-analysis/data
+mca-analysis/tables
+mca-analysis/plots
```

Comment:
Do .gitignore files outside of the root have any effect?

Reply:
they seem to, what I have in those folders after running the scripts is not being pushed. But also, I can move everything to the same .gitignore if it's too chaotic